identity-enabler/verifier-mobile-app/src/pages/Home.svelte

```diff
@@ -14,24 +14,16 @@
 
     const { App, Toast, Modals } = Plugins;
 
-    let isEmpty = false;
     let showTutorial = false;
-    let localCredentials = {};
+    let localCredentials = [];
     let loading = false;
     let exitOnBack = false;
 
     onMount(() => App.addListener("backButton", onBack).remove);
-    onMount(async () => {
-        try {
-            loading = true;
-            localCredentials = await getFromStorage("credentials");
-            localCredentials = Object.values(localCredentials)?.filter(data => data);
-            isEmpty = Object.values(localCredentials).every(x => x === null || x === "");
-            loading = false;
-        } catch (err) {
-            console.error(err);
-            loading = false;
-        }
+    onMount(() => {
+        loading = true;
+        localCredentials = Object.values($credentials).filter(data => Boolean(data));
+        loading = false;
```

Comment:
This `loading` boolean is redundant, see https://github.com/ZebraDevs/Zebra-Iota-Edge-SDK/pull/79  
Will disappear once either PR gets rebased/merged